app/javascript/react/screens/App/Settings/screens/ConversionHostsSettings/components/ConversionHostsList.js

```diff
@@ -1,17 +1,61 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Grid, ListView, Toolbar } from 'patternfly-react';
+import ListViewToolbar from '../../../../common/ListViewToolbar/ListViewToolbar';
 
 const ConversionHostsList = ({ conversionHosts }) => (
-  <React.Fragment>
-    <h2>TODO: render a list view here using the ListViewToolbar pattern we have elsewhere</h2>
-    {conversionHosts.map(conversionHost => (
-      <pre key={conversionHost.id}>{JSON.stringify(conversionHost)}</pre>
-    ))}
-  </React.Fragment>
+  <ListViewToolbar
+    filterTypes={ConversionHostsList.filterTypes}
+    sortFields={ConversionHostsList.sortFields}
+    listItems={conversionHosts}
+  >
+    {({
+      filteredSortedPaginatedListItems,
+      renderFilterControls,
+      renderSortControls,
+      renderActiveFilters,
+      renderPaginationRow
+    }) => (
+      <React.Fragment>
+        <Grid.Row>
+          <Toolbar>
+            {renderFilterControls()}
+            {renderSortControls()}
+            {renderActiveFilters(filteredSortedPaginatedListItems)}
+          </Toolbar>
+        </Grid.Row>
+        <div style={{ overflow: 'auto', paddingBottom: 300, height: '100%' }}>
+          <ListView className="conversion-hosts-list" id="conversion_hosts">
+            {filteredSortedPaginatedListItems.items.map((host, n) => (
+              <ListView.Item key={host.id} heading={host.name} stacked />
+            ))}
+          </ListView>
+          {renderPaginationRow(filteredSortedPaginatedListItems)}
+        </div>
+      </React.Fragment>
+    )}
+  </ListViewToolbar>
 );
 
 ConversionHostsList.propTypes = {
   conversionHosts: PropTypes.arrayOf(PropTypes.object)
 };
 
+ConversionHostsList.sortFields = [
```

Comment:
I was about to comment that we should move these `sortFields` and `filterTypes` into a constants file, but I actually like them here as a property on the component. Unless we end up needing to reuse them somewhere else, this is a fine place for them.

Reply:
Yes, we actually use both approaches in v2v code, I'm fine with going either way with this, just perhaps use same pattern everywhere. What do you think?